sdk/nodejs/s3/bucket.ts

```diff
@@ -4,10 +4,11 @@
 import * as pulumi from "@pulumi/pulumi";
 import * as inputs from "../types/input";
 import * as outputs from "../types/output";
+import * as enums from "../types/enums";
 import * as utilities from "../utilities";
 
 import {PolicyDocument} from "../iam";
-import {CannedAcl} from "./index";
+import {CannedAcl, RoutingRule} from "./index";
```

Comment:
Interesting. It doesn't look like `RoutingRule` is used in this file. Wonder why it's getting pulled in now.

Reply:
Yeah I don't know why this is happening but it's unrelated to the enums. I have another PR where I am just updating the bridge version and seeing the same change. Will clear that up before I merge this.